java/bogdana_maksiuta/src/com/zoolatech/lecture3/tasks/_2/UserAccountValidator.java

```diff
@@ -0,0 +1,140 @@
+package com.zoolatech.lecture3.tasks._2;
+
+public class UserAccountValidator implements Validator {
+    IdUserValidator idUserValidator = new IdUserValidator();
+    NameValidator nameValidator = new NameValidator();
+    LastNameValidator lastNameValidator = new LastNameValidator();
+    CountryValidator countryValidator = new CountryValidator();
+    UserEmailValidator userEmailValidator = new UserEmailValidator();
+    PhoneNumberValid phoneNumberValid = new PhoneNumberValid();
+    BirthDay birthDay = new BirthDay();
+    BirthMonth birthMonth = new BirthMonth();
+
+    @Override
+    public boolean isValid(UserAccount userAccount) {
+        boolean isValid = idUserValidator.isValid(userAccount) && nameValidator.isValid(userAccount)
+                && lastNameValidator.isValid(userAccount) && countryValidator.isValid(userAccount)
+                && userEmailValidator.isValid(userAccount) && phoneNumberValid.isValid(userAccount)
+                && birthDay.isValid(userAccount) && birthDay.isValid(userAccount)
+                && birthMonth.isValid(userAccount);
+        if (isValid) {
+            System.out.println("Access granted");
+        } else {
+            System.out.println("Access denied");
+        }
+        return isValid;
+    }
+
+}
+
+class IdUserValidator implements Validator {
+    public boolean isCorrect(UserAccount userAccount) {
+        System.out.println("Validating that 'id' is correct");
+        return userAccount.getUserId() > 0;
```

Comment:
`You don’t need to perform actual validation, it’s enough to print smth like “validating that fieldName is less than 10” and return true from a smaller validator.`

Reply:
fixed